#Added by TYK
float32 head        # Heading (degree)
float32 rtk_n       # RTK Baseline North coordinate (m)
float32 rtk_e       # RTK Baseline East coordinate (m)
float32 pos_z       # Local Position Z coordinate(m)
float32[3] velocity
